vagrant/provisioning/roles/mariadb/tasks/create_schema.yml

```diff
@@ -14,7 +14,9 @@
         host: "%"
         name: "{{ item }}"
         password: "{{ default_database_password }}" 
-        priv: '*.*:REQUIRESSL/{{ item }}.*:ALL'
+        tls_requires:
+          SSL:
+        priv: '{{ item }}.*:ALL'
```

Comment:
These mysql changes are the only ones that could cause a problem, but these roles aren't used in the upgrade process.